README.md

```diff
@@ -141,7 +163,9 @@ By setting `LOG_STRIPE_SERVICE` to true in your application configuration you ca
 ````javascript
 var ENV = {
   // some vars...
-  LOG_STRIPE_SERVICE: true,
```

Comment:
this was named this way to maintain the Ember standards in env variables, not saying it can't be renamed but giving context into why the choice why made this way and I tend to prefer to adopt existing projects standards.

Reply:
Most Ember addons ship with a namespace under ENV for addon specific config.